phenotypic_profiling_env.yml

```diff
@@ -6,6 +6,9 @@ dependencies:
   - conda-forge::jupyter=1.0.0
   - conda-forge::pandas=1.4.2
   - conda-forge::scikit-learn=1.1.1
+  - conda-forge::umap-learn
+  - conda-forge::fastparquet
+  - conda-forge::pyarrow
```

Comment:
Not sure how this would affect the environment now as well as in the future, but could consider adding version limits